addons/netfox.extras/weapon/network-weapon.gd

```diff
@@ -30,9 +30,9 @@ func fire() -> Node:
 	var data = _projectile_data[id]
 	
 	if not is_multiplayer_authority():
-		rpc_id(get_multiplayer_authority(), "_request_projectile", id, NetworkTime.tick, data)
+		_request_projectile.rpc_id(get_multiplayer_authority(), id, NetworkTime.tick, data)
 	else:
-		rpc("_accept_projectile", id, NetworkTime.tick, data)
+		_accept_projectile.rpc(id, NetworkTime.tick, data)
```

Comment:
Should I undo the latest commit? I just applied #241 and now it shows I did #241 's changes :/